### PR TITLE
resource/alicloud_kvstore_instance: Added the field is_auto_upgrade_open; Updated action TransformToPrePaid to TransformInstanceChargeType to improve the update field payment_type

### PR DESCRIPTION
**Pull request by MrWolong**

### CODE REVIEW

**Review by ChenHanZhang — COMMENTED** [review]
approved

File: alicloud/resource_alicloud_kvstore_instance.go [hunk 1]
```diff
@@ -890,7 +866,83 @@ func resourceAliCloudKvstoreInstanceUpdate(d *schema.ResourceData, meta interfac
...
+	if !d.IsNewResource() && d.HasChange("instance_charge_type") {
+		update = true
+
+		if v, ok := d.GetOk("instance_charge_type"); ok {
+			transformInstanceChargeTypeReq["ChargeType"] = v
+		}
+	}
+
+	if v, ok := d.GetOk("period"); ok {
+		transformInstanceChargeTypeReq["Period"] = v
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_kvstore_instance.go:893]: 这几个应该是属性，从bss接口能取到

Comment (MrWolong) [reply alicloud/resource_alicloud_kvstore_instance.go:893]: 这几个属性是TransformInstanceChargeType的入参，从后付费转为预付费的时候要输入

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

File: alicloud/resource_alicloud_kvstore_instance_test.go [hunk 2]
```diff
@@ -371,6 +371,31 @@ func TestAccAliCloudKVStoreRedisInstance_vpctest(t *testing.T) {
 					}),
 				),
 			},
+			{
```
Comment (MrWolong) [inline alicloud/resource_alicloud_kvstore_instance_test.go:374]: === RUN   TestAccAliCloudKVStoreRedisInstance_vpctest
    provider_test.go:149: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudKVStoreRedisInstance_vpctest (9738.74s)
PASS

**Review by ChenHanZhang — COMMENTED** [review]
approved

File: alicloud/resource_alicloud_kvstore_instance_test.go (hunk 2 above)
Comment (MrWolong) [reply alicloud/resource_alicloud_kvstore_instance_test.go:374]: ci超时报错，查了下日志，都pass了

TF_ACC=1 go test ./alicloud -v -run=TestAccAliCloudKVStoreRedisInstance_5_0_memory_classic_cluster -timeout=1200m
=== RUN   TestAccAliCloudKVStoreRedisInstance_5_0_memory_classic_cluster
    provider_test.go:149: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudKVStoreRedisInstance_5_0_memory_classic_cluster (6991.95s)
PASS
ok  	github.com/aliyun/terraform-provider-alicloud/alicloud	6992.039s

TF_ACC=1 go test ./alicloud -v -run=TestAccAliCloudKVStoreRedisInstance_5_0_memory_classic_standard -timeout=1200m
=== RUN   TestAccAliCloudKVStoreRedisInstance_5_0_memory_classic_standard
    provider_test.go:149: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudKVStoreRedisInstance_5_0_memory_classic_standard (9453.63s)
PASS
ok  	github.com/aliyun/terraform-provider-alicloud/alicloud	9453.721s

TF_ACC=1 go test ./alicloud -v -run=TestAccAliCloudKVStoreRedisInstance_6_0 -timeout=1200m
=== RUN   TestAccAliCloudKVStoreRedisInstance_6_0
--- PASS: TestAccAliCloudKVStoreRedisInstance_6_0 (9447.65s)
PASS
ok  	github.com/aliyun/terraform-provider-alicloud/alicloud	9447.740s

TF_ACC=1 go test ./alicloud -v -run=TestAccAliCloudKVStoreRedisInstance_7_0 -timeout=1200m
=== RUN   TestAccAliCloudKVStoreRedisInstance_7_0
--- PASS: TestAccAliCloudKVStoreRedisInstance_7_0 (4673.85s)
=== RUN   TestAccAliCloudKVStoreRedisInstance_7_0_with_proxy_class
--- PASS: TestAccAliCloudKVStoreRedisInstance_7_0_with_proxy_class (11637.03s)
PASS
ok  	github.com/aliyun/terraform-provider-alicloud/alicloud	16310.964s

TF_ACC=1 go test ./alicloud -v -run=TestAccAliCloudKVStoreRedisInstance_prepaid -timeout=1200m
=== RUN   TestAccAliCloudKVStoreRedisInstance_prepaid
--- PASS: TestAccAliCloudKVStoreRedisInstance_prepaid (7546.27s)
PASS
ok  	github.com/aliyun/terraform-provider-alicloud/alicloud	7546.361s

TF_ACC=1 go test ./alicloud -v -run=TestAccAliCloudKVStoreRedisInstance_vpctest -timeout=1200m
=== RUN   TestAccAliCloudKVStoreRedisInstance_vpctest
    provider_test.go:149: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudKVStoreRedisInstance_vpctest (9850.00s)
PASS
ok  	github.com/aliyun/terraform-provider-alicloud/alicloud	9850.093s

Finish!

**Comment by MrWolong** [comment]
ci超时报错，查了下日志，都pass了

TF_ACC=1 go test ./alicloud -v -run=TestAccAliCloudKVStoreRedisInstance_5_0_memory_classic_cluster -timeout=1200m
=== RUN TestAccAliCloudKVStoreRedisInstance_5_0_memory_classic_cluster
provider_test.go:149: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudKVStoreRedisInstance_5_0_memory_classic_cluster (6991.95s)
PASS
ok github.com/aliyun/terraform-provider-alicloud/alicloud 6992.039s

TF_ACC=1 go test ./alicloud -v -run=TestAccAliCloudKVStoreRedisInstance_5_0_memory_classic_standard -timeout=1200m
=== RUN TestAccAliCloudKVStoreRedisInstance_5_0_memory_classic_standard
provider_test.go:149: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudKVStoreRedisInstance_5_0_memory_classic_standard (9453.63s)
PASS
ok github.com/aliyun/terraform-provider-alicloud/alicloud 9453.721s

TF_ACC=1 go test ./alicloud -v -run=TestAccAliCloudKVStoreRedisInstance_6_0 -timeout=1200m
=== RUN TestAccAliCloudKVStoreRedisInstance_6_0
--- PASS: TestAccAliCloudKVStoreRedisInstance_6_0 (9447.65s)
PASS
ok github.com/aliyun/terraform-provider-alicloud/alicloud 9447.740s

TF_ACC=1 go test ./alicloud -v -run=TestAccAliCloudKVStoreRedisInstance_7_0 -timeout=1200m
=== RUN TestAccAliCloudKVStoreRedisInstance_7_0
--- PASS: TestAccAliCloudKVStoreRedisInstance_7_0 (4673.85s)
=== RUN TestAccAliCloudKVStoreRedisInstance_7_0_with_proxy_class
--- PASS: TestAccAliCloudKVStoreRedisInstance_7_0_with_proxy_class (11637.03s)
PASS
ok github.com/aliyun/terraform-provider-alicloud/alicloud 16310.964s

TF_ACC=1 go test ./alicloud -v -run=TestAccAliCloudKVStoreRedisInstance_prepaid -timeout=1200m
=== RUN TestAccAliCloudKVStoreRedisInstance_prepaid
--- PASS: TestAccAliCloudKVStoreRedisInstance_prepaid (7546.27s)
PASS
ok github.com/aliyun/terraform-provider-alicloud/alicloud 7546.361s

TF_ACC=1 go test ./alicloud -v -run=TestAccAliCloudKVStoreRedisInstance_vpctest -timeout=1200m
=== RUN TestAccAliCloudKVStoreRedisInstance_vpctest
provider_test.go:149: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudKVStoreRedisInstance_vpctest (9850.00s)
PASS
ok github.com/aliyun/terraform-provider-alicloud/alicloud 9850.093s

Finish!

**Pull request merged**